src/disasm.rs

```diff
@@ -1,6 +1,7 @@
 use crate::{
     color,
     demangle::{self, demangled},
+    esafeprintln,
```

Comment:
I think this is not used and will fail CI

Reply:
Whoops, that was from before the changes. Fixed.